src/sqlancer/databend/ast/DatabendBinaryLogicalOperation.java

```diff
@@ -0,0 +1,76 @@
+package sqlancer.databend.ast;
+
+import sqlancer.Randomly;
+import sqlancer.common.ast.BinaryOperatorNode;
+import sqlancer.common.ast.newast.NewBinaryOperatorNode;
+import sqlancer.common.ast.newast.Node;
+
+public class DatabendBinaryLogicalOperation extends NewBinaryOperatorNode<DatabendExpression> {
+
+    public DatabendBinaryLogicalOperation(Node<DatabendExpression> left, Node<DatabendExpression> right,
+            DatabendBinaryLogicalOperator op) {
+        super(left, right, op);
+    }
+
+    public enum DatabendBinaryLogicalOperator implements BinaryOperatorNode.Operator {
+        AND("AND", "and") {
+            // @Override
```

Comment:
I think you can remove all the commented-out code. This is used only by the Pivoted Query Synthesis approach.